iothub/device/src/Properties.cs

```diff
@@ -5,16 +5,20 @@
 using System.Collections;
 using System.Collections.Generic;
 using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
     /// A container for properties.
+    /// <remarks>
+    /// The Properties class is not meant to be constructed by the customer code is meant to be returned from the <see cref="DeviceClient.GetPropertiesAsync(IPayloadConvention, System.Threading.CancellationToken)"/> method.
```

Comment:
punctuation mark seems to be missing in this statement :D

Reply:
Yeah that is missing a word or two too :)